projects/vehicles/worlds/CH_Vens.wbt

```diff
@@ -84,7 +83,8 @@ Fog {
   visibilityRange 8000
 }
 DEF FLOOR Solid {
```

Comment:
It appears to be the case in master already, but the bounding object of the floor is very wrong, being several meters above the ground. Is there perhaps a translation that shouldn't be there or the heights of the actual ground are wrong?

![k](https://user-images.githubusercontent.com/44834743/141069364-f960238c-59b6-4cc5-8838-f5a0a95415fe.png)

Reply:
I opened an issue #3884